recipes/transannot/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "03cc4a7" %}
```

Comment:
Why don't you use a Git tag like https://github.com/soedinglab/transannot/releases/tag/3-7f1c8e1 ?